src/mscorlib/Resources/Strings.resx

```diff
@@ -293,7 +352,7 @@
     <value>Property Get method was not found.</value>
   </data>
   <data name="Arg_GuidArrayCtor" xml:space="preserve">
-    <value>Byte array for GUID must be exactly {0} bytes long.</value>
+    <value>Constructor for Guid needs exactly {0} bytes.</value>
```

Comment:
I don't recall making this change- is it possible that changes to this file are automatically generated? I'll revert the changes like you suggested.